x-pack/plugins/enterprise_search/public/applications/index.tsx

```diff
@@ -67,6 +68,7 @@ export const renderApp = (
       >
         <LicenseProvider license$={plugins.licensing.license$}>
           <Provider store={store}>
+            <HttpProvider http={core.http} errorConnecting={errorConnecting} />
```

Comment:
Just wanted to make a quick note here: I left `http` in KibanaContext for now:

https://github.com/elastic/kibana/blob/94b5da6ae91fe2a3bf30a1c1945bc4bc8a1b061c/x-pack/plugins/enterprise_search/public/applications/index.tsx#L59-L63

although I'm 50/50 on whether to remove it and have the only way for us to access http be from `const { http } = useValues(HttpLogic)` (vs also being able to do `const { http } = useContext(KibanaContext)`).

They both reference the same global Kibana http instance so it's not a question of accidentally using the wrong var or anything, it's just 2 different ways of calling `http` which may end up being confusing.

I will say that IMO `useContext(KibanaContext)` is significantly easier to mock and spy on in Jest tests (you just listen for `mockKibanaContext.http` changes) - whereas I ran into a super huge headache trying to pass a mockContext into a HttpLogic mock for some baffling reason.

Either way we decide, I definitely don't plan on removing all instances of `const { http } = useContext(KibanaContext)` in this PR, to try to keep it smaller for now. @scottybollinger / @JasonStoltz would definitely appreciate your thoughts if we should eventually remove them or if it's not a huge deal to have 2 ways of accessing `http`.

Reply:
I personally don't mind leaving both instance types since they reference the same thing and especially if it makes testing easier

Reply:
Yeah I'm 👍 for just leaving it as is for now and seeing how it goes. We could always change it later.